CSLabs.Api/Startup.cs

```diff
@@ -63,6 +65,23 @@ public void ConfigureServices(IServiceCollection services)
             ConfigureJWT(services, appSettings.JWTSecret);
             services.AddAutoMapper(Assembly.GetExecutingAssembly());
             services.ProvideAppServices();
+            
+            // Add Hangfire services.
+            services.AddHangfire(configuration => configuration
+                .SetDataCompatibilityLevel(CompatibilityLevel.Version_170)
+                .UseSimpleAssemblyNameTypeSerializer()
+                .UseRecommendedSerializerSettings()
+                .UseSqlServerStorage(Configuration.GetConnectionString("HangfireConnection"), new SqlServerStorageOptions
```

Comment:
This Looks like it's using the wrong database engine, we aren't using SqlServer, We're using MariaDB.